msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Format AI bot messages
+description: Learn how to format your AI-based bot with an AI label, sensitivity labels, citations, and feedback buttons.
+ms.topic: conceptual
+ms.localizationpriority: medium
+---
+
+# Format AI bot messages
+
+If you're building a conversational bot, you can now leverage features such as citations and feedback buttons that enable better user engagement. For AI-powered bots, you can now add AI label and sensitivity label that can help you build trust and transparency with users.
```

Comment:
If (some of?) these are applicable to all bots, why lump them into a "Format AI bot messages" node? It might be better to introduce them as generic features for bot messages, and then in this article discuss the specific use cases and recommended best practices for AI bots.

Reply:
The title and introductory paragraph have now been updated to eliminate the confusion.